ui/src/components/providers.tsx

```diff
@@ -17,12 +17,12 @@
  * License-Filename: LICENSE
  */
 
+import { OpenAPI as OpenAPIConfig } from '@/api/requests/core/OpenAPI';
+import { Toaster } from '@/components/ui/toaster';
 import { queryClient } from '@/lib/query-client';
 import { QueryClientProvider } from '@tanstack/react-query';
 import { ReactNode } from 'react';
 import { AuthProvider, AuthProviderProps } from 'react-oidc-context';
-import { Toaster } from '@/components/ui/toaster';
-import { OpenAPI as OpenAPIConfig } from '@/api/requests/core/OpenAPI';
```

Comment:
Just an example but this happens across the files. The sorting does not work, as it considers the imports from local code with the alias `@/` to be third-party code. Not sure what the best way to fix it is, but the differentiator is that third party code from namespaces doesn't have `/` after `@`.